hack/tests/subcommand-scorecard.sh

```diff
@@ -41,10 +41,10 @@ else
 fi
 
 header_text 'scorecard test to see if bundle flag works correctly'
-if ! commandoutput="$(operator-sdk scorecard --config "$CONFIG_PATH_BUNDLE" 2>&1)"; then 
+if ! commandoutput="$(operator-sdk scorecard --config "$CONFIG_PATH_BUNDLE" 2>&1)"; then
 	echo $commandoutput
 	failCount=`echo $commandoutput | grep -o "fail" | wc -l`
-	expectedFailCount=3
+	expectedFailCount=2
```

Comment:
Since the `bundle` key is directly used instead of parsing it and all adjacent bundles, the `NotExistKind` error is no longer encountered by design.

Reply:
Thanks for removing the trailing spaces!